### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -74,19 +75,6 @@ if [ -n "$modified" ]; then
...
-  echo "You have local changes to files/directories that are in your git ignore list."
-  echo "These need to be removed, as they may interfere with the build. Release builds"
-  echo "need to be clean builds. To clean your directory, run:"
-  echo
-  echo "  git -C '$(pwd)' clean -fdx"
-  echo
-  echo "This will have the following effect:"
-  git clean -ndx
-  exit 70
-fi
```
Comment (petemoore) [inline release.sh]: Yeah, I guess it is good to remove this, perhaps it was overkill, as we don't build locally.

File: release.sh [hunk 2]
```diff
@@ -112,6 +100,7 @@ git add README.md
 git commit -m "Version bump from ${OLD_VERSION} to ${NEW_VERSION}"
 git tag -s "v${NEW_VERSION}" -m "Making release ${NEW_VERSION}"
 # only ensure master is updated if it is a non-alpha release
+exit
```
Comment (petemoore) [inline release.sh]: I don't think we want this `exit` here.

File: release.sh [hunk 3]
```diff
@@ -57,7 +57,8 @@ function inline_sed {
 # Make sure git tag doesn't already exist on remote
 if [ "$(git ls-remote -t "${OFFICIAL_GIT_REPO}" "v${NEW_VERSION}" 2>&1 | wc -l | tr -d ' ')" != '0' ]; then
   echo "git tag '${NEW_VERSION}' already exists remotely on ${OFFICIAL_GIT_REPO},"
-  echo "or there was an error checking whether it existed."
+  echo "or there was an error checking whether it existed:"
+  git ls-remote -t "${OFFICIAL_GIT_REPO}" "v${NEW_VERSION}"
```
Comment (petemoore) [inline release.sh]: Nice.

File: mozilla-try-scripts/gecko-try.sh [hunk 4]
```diff
@@ -121,14 +121,16 @@ for MANIFEST in *-b.json *-cu.json *-beta.json; do
...
 git add .
 git commit -m "Testing generic-worker ${NEW_GW_VERSION} / taskcluster-proxy ${NEW_TP_VERSION} on *STAGING*
 
 This change does _not_ affect any production workers. Commit made with:
 
-    ${0} ${@}
+    ${0} $(echo ${@})
 
-See https://github.com/taskcluster/generic-worker/blob/$(git -C "${THIS_SCRIPT_DIR}" rev-parse HEAD)/$(git -C "${THIS_SCRIPT_DIR}" ls-files --full-name "$(basename "${0}")")" -m "${DEPLOY}"
+See https://github.com/taskcluster/generic-worker/blob/$THIS_REV/$THIS_FILE -m \"${DEPLOY}\""
```
Comment (petemoore) [inline mozilla-try-scripts/gecko-try.sh]: Nice.

File: mozilla-try-scripts/gecko-try.sh [hunk 5]
```diff
@@ -121,14 +121,16 @@ for MANIFEST in *-b.json *-cu.json *-beta.json; do
...
 DEPLOY="deploy: $(git status --porcelain | sed -n 's/^ M userdata\/Manifest\/\(.*\)\.json$/\1/p' | tr '\n' ' ')"
+THIS_REV="$(git -C "${THIS_SCRIPT_DIR}" rev-parse HEAD)"
+THIS_FILE="$(git -C "${THIS_SCRIPT_DIR}" ls-files --full-name "$(basename "${0}")")"
 git add .
 git commit -m "Testing generic-worker ${NEW_GW_VERSION} / taskcluster-proxy ${NEW_TP_VERSION} on *STAGING*
 
 This change does _not_ affect any production workers. Commit made with:
 
-    ${0} ${@}
+    ${0} $(echo ${@})
```
Comment (petemoore) [inline mozilla-try-scripts/gecko-try.sh]: Thanks for working out the fix for this! Not sure why we didn't hit this on macOS, but as we get it on linux, I'm happy to have the fix.

File: release.sh (hunk 2 above)
Comment (djmitche) [reply release.sh]: Oh no!  It was to prevent any accidents :)

**Pull request merged**